### DISCUSSION TITLE
Anthropic: Garbled Characters in Chinese Outputs

### DISCUSSION THREAD

**Discussion by RedwindA** [danny-avila/LibreChat#3834] [Troubleshooting]
### What happened?

similar to https://github.com/danny-avila/LibreChat/issues/1609

it's NOT Anthropic's offficial endpoint. I'm using an external tool that converts vertex's claude into Anthropic API format.

### Steps to Reproduce

Ask questions in Chinese, and then notice the appearance of � in the responses

### What browsers are you seeing the problem on?

Microsoft Edge

### Screenshots

![image](https://github.com/user-attachments/assets/50165abf-c891-47b3-8b1f-faef83771ebc)

The message in mongodb：
![image](https://github.com/user-attachments/assets/36c4a5d4-94fe-4fd8-8186-1d8f69ef5643)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
i can't observe this with the official Anthropic API:

https://github.com/user-attachments/assets/409cf289-90a2-4725-897f-827ea49599d6


https://github.com/user-attachments/assets/6d997b1f-db4d-4fbf-8cae-bf71aa1c1940

**Reply by danny-avila** [reply]
Seems Vertex is the common denominator? Need to fix that issue there still: https://github.com/danny-avila/LibreChat/pull/3427
